src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	public String getPrintableString(Printable... other);
```

Comment:
Wrong method implemented. it should be public String getPrintableString();
In each class which implements Printable interfaces return the respectable String i.e. Name: John
Then in the Person.java class you implement a separate method call public String getPrintableString(Printable... other) which will print all the details.
Hope the exercise is clear to you now. I think you have mixed it up a bit here. Good attempt though.